doc/mock_example/input.yaml

```diff
@@ -1,10 +1,41 @@
-# mock target configuration file
+contaminants:
+    galaxies:
+        Buzzard: {
+            mockfile: '{DESI_ROOT}/mocks/buzzard/buzzard_v1.6_desicut',
+            nside_buzzard: 8,
+            format: buzzard,
+        }
+    stars:
+        MWS_MAIN: {
+            mockfile: '{DESI_ROOT}/mocks/mws/galaxia/alpha/v0.0.6/healpix',
+            nside_galaxia: 8,
+            format: galaxia,
+            FAINTSTAR: {
+                mockfile: '{DESI_ROOT}/mocks/mws/galaxia/alpha/v0.0.6_superfaint/healpix',
+                magcut: 23.5,
+            }
+        }
+    targets:
+        BGS: {
+            stars: 0.05,
+        }
+        ELG: {
+            stars: 0.05,
+            galaxies: 0.1,
+        }
+        LRG: {
+            stars: 0.04,
+        }
+        QSO: {
+            stars: 0.25,
+            galaxies: 0.1,
+        }
```

Comment:
Please document with a comment in the yaml file what these numbers are.

Are these fractions of targets to treat as contaminants of the particular types?  If so, should that concept be owned somewhere in desimodel/data/targets/ instead along with densities and n(z)?  I recognize that updates to desimodel are more of a hassle, so it could be reasonable to keep contaminant fractions here until our understanding has stabilized, and then move to desimodel.  For consideration.

Reply:
These are indeed fractions.  I agree with the comment in principle, but until we have a set of mocks that are specifically for contaminants anything I do is going to be a hack.  I propose we move forward with what I'm implementing---which we'll be able to evaluate much more easily in the past because I'm adding more QA to this PR---and then go from there.